app/views/components/_content-metadata.html.erb

```diff
@@ -0,0 +1,18 @@
+<%
+  published ||= false
+  last_updated ||= false
+  from ||= false
+  metadata = [published, last_updated, from]
+%>
+<% if metadata.any? %>
+  <div class="app-c-content-metadata">
+    <div class="grid-row">
```

Comment:
Not sure how the metadata is used on pages, but at the moment it looks like it's all inside this grid/column? Does that mean we could take it out of the component and wrap the component inside it when needed in the layouts?

Reply:
The grey borders need to run full width. So I think having the internal grid is ok.

Reply:
I don't think this is a blocker but I think this'd make more sense without and using `max-width` instead.

This grid is being used to make sure the [line length](https://en.wikipedia.org/wiki/Line_length) a good length to make the content legible.